packages/calcite-components/src/components/panel/panel.e2e.ts

```diff
@@ -424,4 +425,127 @@ describe("calcite-panel", () => {
     expect(await panel.getProperty("closed")).toBe(false);
     expect(await container.isVisible()).toBe(true);
   });
+
+  describe("theme", () => {
+    const panelHTML = html`
+      <calcite-panel height-scale="s" closable heading="panel" description="slotted with header actions and action-bar">
+        <calcite-action text="banana" text-enabled icon="banana" slot="header-menu-actions"></calcite-action>
+        <calcite-action text="measure" text-enabled icon="measure" slot="header-menu-actions"></calcite-action>
+        <calcite-action label="icon left" icon="left" slot="header-actions-start"></calcite-action>
+        <calcite-action label="icon left" icon="left" slot="header-actions-end"></calcite-action>
+        <calcite-action-bar slot="action-bar">
+          <calcite-action-group>
+            <calcite-action text="Add" icon="plus"> </calcite-action>
+            <calcite-action text="Save" icon="save"> </calcite-action>
+            <calcite-action text="Layers" icon="layers"> </calcite-action>
+          </calcite-action-group>
+        </calcite-action-bar>
+        <div slot="content-top">Slot for a content-top.</div>
+        <p>Slotted content!</p>
+        <p style="height: 400px">Hello world!</p>
+        <p style="height: 400px">Hello world!</p>
+        <p style="height: 400px">Hello world!</p>
+        <div slot="content-bottom">Slot for a content-bottom.</div>
+        <p slot="footer">Slotted content!</p>
+        <calcite-fab slot="fab"></calcite-fab>
+      </calcite-panel>
+    `;
+
+    describe("default", () => {
+      const tokens: ComponentTestTokens = {
+        "--calcite-panel-spacing-block-end": {
+          shadowSelector: `.${CSS.contentBottom}`,
+          targetProp: "paddingBlockEnd",
+        },
+        "--calcite-panel-spacing-block-start": {
+          shadowSelector: `.${CSS.contentTop}`,
+          targetProp: "paddingBlockStart",
+        },
+        "--calcite-panel-height": {
+          selector: "calcite-panel",
+          targetProp: "blockSize",
+        },
+        "--calcite-panel-width": {
+          selector: "calcite-panel",
+          targetProp: "inlineSize",
+        },
+        "--calcite-panel-action-background-color": {
+          shadowSelector: `calcite-action >>> .button`,
```

Comment:
I think this can just be "calcite-action"

Reply:
I think there is a confusion with naming here. The `--calcite-panel-action-background-color` token is only used for close action rendered when `closable` is set to `true` & the trigger of calcite-action-menu when header-menu-actions are slotted.

Should we separate these out and refactor names to be more specific ?